main.ts

```diff
@@ -248,10 +249,27 @@ export default class Moviegrabber extends Plugin {
 		}
 
 		this.createNote(itemData, type, path);
```

Comment:
i think this needs to be calles after the Posterimage save.

Reply:
no doesn't matter

Reply:
but then the posterimage data will not be writen to the itemData and will not be available in the template or am i wrong?

Reply:
Misunderstanding